ingestion/src/metadata/ingestion/source/pipeline/dbtcloud/models.py

```diff
@@ -35,7 +35,7 @@ class DBTJob(BaseModel):
 
 
 class DBTJobList(BaseModel):
-    Jobs: Optional[List[DBTJob]] = Field([], alias="data")
+    Jobs: List[DBTJob] = Field(alias="data")
```

Comment:
Is this not Optional anymore?

Reply:
yes. Its for validating that we should be getting `data` | `Jobs` value otherwise no reason to proceed further. 
As we would be ingesting those `Jobs` as pipeline.